hibernate-core/src/main/java/org/hibernate/metamodel/model/domain/internal/JpaMetamodelImpl.java

```diff
@@ -487,8 +499,6 @@ public void processJpa(
 
 			context.wrapUp();
 
-			this.nameToImportNameMap.putAll( bootMetamodel.getImports() );
```

Comment:
@sebersole it seems the above statement is redundant for it has been done at the beginning of the method. Is that correct?